templates/navigation/oe-corporate-blocks-ec-footer--core.html.twig

```diff
@@ -21,7 +21,7 @@
       'path': path,
       'src_desktop': desktop_logo_path,
     },
-    'description': site_owner ? 'This site is managed by: @name'|t({'@name': site_owner}),
+    'description': site_owner ? 'This site is managed by:<br />@name'|t({'@name': site_owner}),
```

Comment:
It will make the translation of the string not applicable. Translation should be updated.